advanced/option-usebadger.rst

```diff
@@ -1,3 +1,5 @@
+.. _advanced-option-usebadger:
+
```

Comment:
Wasn't this support even removed lately?
https://github.com/syncthing/syncthing/pull/7413